src/components/VMenu/mixins/menu-activator.js

```diff
@@ -21,11 +21,15 @@ export default {
   methods: {
     activatorClickHandler (e) {
       if (this.disabled) return
-      else if (this.openOnClick && !this.isActive) {
+      if (this.openOnClick && !this.isActive) {
+        this.$slots.activator[0].elm.focus()
```

Comment:
What happens if this.$slots.activator is empty?

Reply:
Fixed, I didn't realise that there was an `activator` prop. We really need to figure out how to get mixins into the docs.